commune_app/tests/test_chores.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from commune_app import models
+from commune_app.all_models.users import User
+from commune_app.all_models.chores import Chore
+
+
+USER1_ID = 206609999
+USER1_USERNAME = "Elias"
+USER1_PASSWORD = "Wtf123"
+USER1_FIRST_NAME = "Elias"
+USER1_LAST_NAME = "Assy"
+USER1_EMAIL = "eliasakahaxor@gmail.com"
+
+CHORE_TITLE = "Dishes"
+CHORE_DATE = (1997, 10, 19)
+CHORE_BUDGET = 30
+CHORE_DESCRIPTION = "Clean the Dishes"
+CHORE_ASSIGN = "Elias"
+CHORE_PASSED = True
+
+
+@pytest.fixture
+def user0():
+    return User(
+        id=USER1_ID,
+        username=USER1_USERNAME,
+        password=USER1_PASSWORD,
+        first_name=USER1_FIRST_NAME,
+        last_name=USER1_LAST_NAME,
+        email=USER1_EMAIL
+    )
+
+
+@pytest.fixture
+def chore0(user0):
+    return models.Chore(
+        title=CHORE_TITLE,
+        date=CHORE_DATE,
+        budget=CHORE_BUDGET,
+        description=CHORE_DESCRIPTION,
+        assign=user0,
+        passed=CHORE_PASSED
+    )
+
+
+@pytest.fixture
+def persist_chore(db, chore0):
+    chore0.save()
+    return chore0
+
+
+@pytest.mark.django_db()
+class TestChoreModel:
+
+    def test_delete_chore(self, persist_chore):
+        persist_chore.delete()
+        assert persist_chore not in Chore.objects.all()
+
+    def test_new_chore(self, chore0):
+        assert chore0.title == CHORE_TITLE
+        assert chore0.date == CHORE_DATE
+        assert chore0.budget == CHORE_BUDGET
+        assert chore0.description == CHORE_DESCRIPTION
+        assert chore0.assign.first_name == "Elias"
+        assert chore0.passed == CHORE_PASSED
+
+    def create_chore(self, title, date, budget, assign, description, passed):
```

Comment:
maybe you can add here a matrix of tests with parameterize as you can see for example in:
https://github.com/redhat-beyond/commune/blob/main/commune_app/tests/test_user.py#L74 
```suggestion
    def test_create_chore(self, title, date, budget, assign, description, passed):
```

Reply:
if i change it to test_create_chore ill get an error in pytest..
also i made it similar to the example you guys showed us in this link below:

https://github.com/redhat-beyond/lets-meet/blob/main/users/class_tests/users_tests.py#L42

as you can see in line 39 he didnt use test..

Reply:
as you can see in:
https://github.com/redhat-beyond/lets-meet/blob/main/users/class_tests/users_tests.py#L68
they use it as part of the test
    def test_invalid_user_values(self, email, phone_num, password, username, excpected_error):
and all the parameters they got from the pytest parameterize

Reply:
i tried same as the user test and the other link and i had many issues so ended up deleting it.. 
after merge and all ill try to fix it if its needed